media/style.css

```diff
@@ -34,5 +50,73 @@
 .divider {
   width: 100%;
   border-bottom: 1px solid rgba(0, 0, 0, 0.5);
-  margin: 20px 0px;
+  margin: 20px 0;
+}
+
+/* -------- ATTRIBUTES -------- */
```

Comment:
I think we can add these to `media/schema-manager/schema-details.css` wdyt? To keep `style.css` feature-agnostic

Reply:
lets keep all css together because for now, it makes no sense to separate it due to we need to import both CSS  files